src/main/java/battlestates/states/EnemyTurnState.java

```diff
@@ -0,0 +1,57 @@
+package battlestates.states;
+
+import interfaces.State;
+import io.InputValidator;
+import objects.battle.StatDisplayer;
+import objects.battle.enemy.ai.EnemyActionHandler;
+import objects.character.EnemyFacade;
+import objects.character.Player;
+
+public class EnemyTurnState implements State {
+    private Player user;
+    private EnemyFacade foe;
+    private boolean done = false;
+    private String userAction;
+
+    public EnemyTurnState(Player user, EnemyFacade foe, String userAction) {
```

Comment:
Depending on how user inputs are, we might want talk a little about how the inputs are formatted so that enemyAction() in preInput method works as intended.